### PR TITLE
Fix fungible README hashes

### PR DESCRIPTION
**Pull request by andresilva91**
## Motivation

A change in how the PRNG generates these hashes happened, and we didn't update it here

## Proposal

Updating hashes

## Test Plan

CI

## Follow ups

Will write a graphql test in the `README`, and maybe also an UI test, to make sure this is caught by CI

### CODE REVIEW

**Comment by andresilva91** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/linera-io/linera-protocol/1734?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

Current dependencies on/for this PR:
* **#1733** <a href="https://app.graphite.dev/github/pr/linera-io/linera-protocol/1733?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#1734** <a href="https://app.graphite.dev/github/pr/linera-io/linera-protocol/1734?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#1725** <a href="https://app.graphite.dev/github/pr/linera-io/linera-protocol/1725?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/linera-io/linera-protocol/1734?utm_source=stack-comment).

**Review by afck — COMMENTED** [review]
That hash also occurs in the counter, matching-engine and amm READMEs.

File: examples/amm/README.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ We use the test-only CLI option `--testing-prng-seed` to make keys deterministic
 explanation.
 
 ```bash
-OWNER_1=e814a7bdae091daf4a110ef5340396998e538c47c6e7d101027a225523985316
+OWNER_1=7136460f0c87ae46f966f898d494c4b40c4ae8c527f4d1c0b1fa0f7cff91d20f
 OWNER_2=453690095cdfe6dbde7fc577e56bb838a7ee7920a72512d4a87748b4e151ed61
```
Comment (afck) [inline examples/amm/README.md:55]: Probably owner 2 has also changed?

Comment (andresilva91) [reply examples/amm/README.md]: Ah, right. Good catch. I wonder why the README CI isn't catching it though, since we have graphQL tests 🤔

**Review by jvff — APPROVED** [review]

**Comment by andresilva91** [comment]
### Merge activity

* **Mar 1, 12:18 PM EST**: @andresilva91 started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/linera-io/linera-protocol/1734).

**Pull request merged**